src/or/connection.c

```diff
@@ -3117,7 +3121,8 @@ connection_consider_empty_read_buckets(connection_t *conn)
   if (!connection_is_rate_limited(conn))
     return; /* Always okay. */
 
-  bool is_global = true;
+  int is_global = 1;
+
```

Comment:
What is going here? Actually, do we need this `is_global` at all? It is still ignored in the `*_bw_exhausted()` functions.

Reply:
I'd like to leave it in, since we will likely want it in the future if we ever have special handling for the per-connection buckets.

Reply:
Ok but then why the removal of the `bool` ?